docs/source/llms/llama-index/index.rst

```diff
@@ -221,6 +224,62 @@ You can disable tracing by running the same function with the ``disable`` parame
 FAQ
 ---
 
+
+How to log and load an index with external vector stores?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your index uses the default ``SimpleVectorStore``, you can log the index directly to MLflow using the :py:func:`mlflow.llama_index.log_model` function. MLflow persists the in-memory index data (embedded documents) to MLflow artifact store, which allows loading the index back with the same data without re-indexing the documents.
+
+However, when the index uses external vector stores like ``DatabricksVectorSearch`` and ``QdrantVectorStore``, the index data is stored remotely and they do not support local serialization. Thereby, you cannot log the index with these stores directly. For such cases, you can use the `Model-from-Code <../../../models.html#models-from-code>`_ logging that provides more control over the index saving process and allow you to use the external vector store.
+
+To use model-from-code logging, you first need to create a separate Python file that defines the index. If you are on Jupyter notebook, you can use the ``%%writefile`` magic command to save the cell code to a Python file.
+
+.. blacken-docs:off
+
+.. code-block:: python
+
+    %%writefile index.py
+
+    # Create Qdrant client with your own settings.
+    client = qdrant_client.QdrantClient(
+        host="localhost",
+        port=6333,
+    )
```

Comment:
To run this code, do I need to set up something?

Reply:
We need to start a Qdrant server as a docker container. https://qdrant.tech/documentation/quickstart/. It may be cumbersome for ppl who hasn't set up docker.